src/Components/WebAssembly/Build/testassets/rclwithnodependencies/Class2.cs

```diff
@@ -0,0 +1,6 @@
+namespace RazorClassLibrary
```

Comment:
If it's possible to remove the `.cs`/`.css`/`.js` files from the test RCLs and it still works correctly, then that would be ideal. AFAICT the `.js`/`.css` at least shouldn't be affecting the behavior here.

Reply:
Done.